### DISCUSSION TITLE
Welcome to Discussions!

### DISCUSSION THREAD

**Discussion by DamieFC** [Platypus-Tech/platypus-os-original-drafts#5] [General]
## Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together. Also, please follow the Code Of Conduct.

**Comment by TIBTHINK** [comment]
do you have a discord i can dm you?

**Reply by DamieFC** [reply]
No, I only use Google Hangouts.
